document-generator-accounts/src/main/java/uk/gov/companieshouse/document/generator/accounts/data/accounts/AccountsManager.java

```diff
@@ -299,6 +300,45 @@ public SmallFullAccountIxbrl getSmallFullAccounts(String link, Transaction trans
                 smallFullApiData.setOffBalanceSheet(offBalanceSheetApi);
             }
 
+            if (!StringUtils.isEmpty(smallFull.getLinks().getLoansToDirectors())) {
```

Comment:
I wonder if we could start to reduce the bloat of this method by moving the internals of these if methods into one-off private methods? Starting with this one?

Reply:
Good shout - improves legibility. Done in [e2d7aad](https://github.com/companieshouse/document-generator/pull/228/commits/e2d7aadcae5cc7e2bbbb09796acbc31c5bd157ac)